cloudigrade/util/tests/test_aws_helper.py

```diff
@@ -96,6 +108,12 @@ def assert_verify_policy_action_success(self, action, function_name,
             func_kwargs (dict): keyword arguments that would be sent to the ec2
                 method called by _verify_policy_action
         """
+        cloudtrail_actions = ['cloudtrail:DescribeTrails',
```

Comment:
It looks like this list is duplicated from `util.aws.helper`. Delete and import directly from that module for consistency, or if you go the route of the `startswith` style suggestion change the test accordingly.